playbooks/responses/not-an-issue.md

```diff
@@ -1 +1,3 @@
-GitHub issues are for feature requests and bug reports, questions about using Electron or code assistance requests should be directed to the [community](https://github.com/electron/electron#community).
+The Electron team uses its issues for tracking feature requests and bug reports.
+
+Please visit https://github.com/electron/electron#community for a list of resources that are better oriented for questions and help. Thank you!
```

Comment:
:grimacing: the [transitive link](https://github.com/electron/electron/blob/v12.0.0-nightly.20201021/docs/tutorial/support.md) here lists the Atom Slack, not the Discord server...

We should fix that, but FWIW I've been using this link instead:
```suggestion
Please visit https://www.electronjs.org/community for a list of resources that are better oriented for questions and help. Thank you!
```

Reply:
Link fixed in cdbc8549daa5107d9d18f256bfb5e53cce65eef9.